edxval/tests/test_api.py

```diff
@@ -174,7 +174,8 @@ def test_create_external_video(self):
         }
         edx_video_id = api.create_external_video(display_name=expected_video['client_video_id'])
         video = VideoSerializer(Video.objects.get(edx_video_id=edx_video_id)).data
-        self.assertDictContainsSubset(expected_video, video)
+        for item in expected_video.items():
+            assert item in video.items()
```

Comment:
The original assertion also checked that the values match in both dicts for keys in the subset dict.  I've seen a few different suggestions for replacing this deprecated method, I think the cleanest I've seen so far is at https://github.com/pytest-dev/pytest/issues/2376#issuecomment-496661974

Reply:
updated. items() returns the tuple (key, value) and when comparing both values are checked.